src/hooks/queries/useEvmFees.ts

```diff
@@ -15,6 +15,7 @@ import { useAppSelector } from 'state/store'
 
 type UseEvmFeesProps = {
   accountNumber: number | undefined
+  pubKey: string | undefined
```

Comment:
I agree we should do it for `adapter.getAddress()` too, can we create an issue so we track this? Forgetting this could lead to some new bugs introduced for ledger, it seems important to change this from my point of view